src/__tests__/get-local-project-data.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { getLocalProjectData } from '@/utils/get-local-project-data';
```

Comment:
I think we are standardizing moving tests side by side with the actual file.

Can you move this please.